Formula/l/llvm@18.rb

```diff
@@ -0,0 +1,709 @@
+class LlvmAT18 < Formula
+  desc "Next-gen compiler infrastructure"
+  homepage "https://llvm.org/"
+  url "https://github.com/llvm/llvm-project/releases/download/llvmorg-18.1.8/llvm-project-18.1.8.src.tar.xz"
+  sha256 "0b58557a6d32ceee97c8d533a59b9212d87e0fc4d2833924eb6c611247db2f2a"
+  # The LLVM Project is under the Apache License v2.0 with LLVM Exceptions
+  license "Apache-2.0" => { with: "LLVM-exception" }
+
+  livecheck do
+    url :stable
+    regex(/^llvmorg[._-]v?(\d+(?:\.\d+)+)$/i)
+  end
+
+  # Clang cannot find system headers if Xcode CLT is not installed
+  pour_bottle? only_if: :clt_installed
+
+  keg_only :versioned_formula
+
+  # https://llvm.org/docs/GettingStarted.html#requirement
+  depends_on "cmake" => :build
+  depends_on "ninja" => :build
+  depends_on "swig" => :build
+  depends_on "python@3.12"
+  depends_on "xz"
+  depends_on "z3"
+  depends_on "zstd"
+
+  uses_from_macos "libedit"
+  uses_from_macos "libffi", since: :catalina
+  uses_from_macos "ncurses"
+  uses_from_macos "zlib"
+
+  on_linux do
+    depends_on "pkg-config" => :build
+    depends_on "binutils" # needed for gold
+    depends_on "elfutils" # openmp requires <gelf.h>
+  end
+
+  # Fails at building LLDB
+  fails_with gcc: "5"
+
+  def python3
+    "python3.12"
+  end
+
+  def install
+    # The clang bindings need a little help finding our libclang.
+    inreplace "clang/bindings/python/clang/cindex.py",
+              /^(\s*library_path\s*=\s*)None$/,
+              "\\1'#{lib}'"
+
+    projects = %w[
+      clang
+      clang-tools-extra
+      lld
+      mlir
+      polly
+    ]
+    runtimes = %w[
+      compiler-rt
+      libcxx
+      libcxxabi
+      libunwind
+    ]
+
+    unless versioned_formula?
+      projects << "lldb"
+
+      if OS.mac?
+        runtimes << "openmp"
+      else
+        projects << "openmp"
+      end
+    end
```

Comment:
This is dead code and should be removed.

Reply:
```suggestion
```

Reply:
I can remove this lines, of course, but they were already in llvm.rb for version 18. Won't removing them make it more difficult to demote 19 to a versioned formula (requiring adding these lines back in) when 20 comes out and so on every 6 months?

Reply:
Ah, sorry, I see this was llvm@18, not llvm.rb. I'll get it removed.